docs/Contribute/README.md

```diff
@@ -1,14 +1,16 @@
 # How to contribute to BTCPay Server?
 
 BTCPay Server is built and maintained entirely by volunteer contributors around the internet.
-We welcome and appreciate new contributions, whether you're a developer, a five language speaking individual or an average Joe or Jane that wants to help in any way possible.
 
+We welcome, appreciate and encourage new contributions. 
 
-Contributors can help in a number of ways:
-* [Coding for BTCPay Server](../Contribute/ContributeDev/README.md)
-* [Writing for BTCPay Server](../Contribute/ContributeWrite/README.md)
-* [Designing for BTCPay Server](ContributeDesign.md)
-* [Translating for BTCPay Server](ContributeTranslate.md)
+Depending on your skills and interest, you can help in a number of ways:
+
+* [Code and development](../Contribute/ContributeDev/README.md)
+*  [Testing](../ContributeDevTest.md)
```

Comment:
```suggestion
* [Testing](../ContributeDevTest.md)
```

Reply:
It's not, it should create a sub-bullet

Reply:
I'm pretty sure it won't.
Should be a `Tab` before the bullet point.